### ISSUE TITLE
rss2email: Inline stylesheet not rendered in Rainloop

### ISSUE THREAD

**Issue by megadr01d** [RainLoop/rainloop-webmail#1495]
**RainLoop version, browser, OS:**
1.11.1, Chrome/Safari, Windows/OS X

**Expected behavior and actual behavior:**
Using [rss2email](https://github.com/turbodog/rss2email) (command line util) to deliver RSS feeds to email and using a custom [stylesheet](https://github.com/turbodog/rss2email/blob/686d5a3bee38f9aa8d6b6e0d6b221d926d020620/config.py.example#L87) when delivering the feeds as HTML, fails to render the stylesheet in Rainloop. For reference, it works in Roundcube and local OS X Mail client.

I noticed rss2email renders the stylesheet [inline](https://github.com/turbodog/rss2email/blob/686d5a3bee38f9aa8d6b6e0d6b221d926d020620/rss2email.py#L729), if that makes a difference.

**Steps to reproduce the problem:**
N/A short of having to install and configure rss2email.

**Logs or screenshots:**
Rainloop

![rainloop-msg-top](https://user-images.githubusercontent.com/9649385/29921426-742cdb40-8e49-11e7-9a91-6be1f0f1e9f1.png)
![rainloop-msg-middle](https://user-images.githubusercontent.com/9649385/29921431-7861b118-8e49-11e7-885b-66ad97801e1c.png)
![rainloop-msg-bottom](https://user-images.githubusercontent.com/9649385/29921436-7aef6e5c-8e49-11e7-9eb6-e3a22d13dbc3.png)

Roundcube

![roundcube-msg-top](https://user-images.githubusercontent.com/9649385/29920845-56d82ee8-8e47-11e7-88f4-5ff19af489a8.png)
![roundcube-msg-middle](https://user-images.githubusercontent.com/9649385/29920851-5ce6b142-8e47-11e7-9d71-bd8d12e5cb1c.png)
![roundcube-msg-bottom](https://user-images.githubusercontent.com/9649385/29920856-61170afa-8e47-11e7-8de8-9e4400bb6430.png)

**Comment by megadr01d** [comment]
Testcase `eml` files. Some feesd are encoded in base64 but in the 7bit ones, I noticed the stylesheet is commented out:

```
<head><style><!--body {font-family: Helvetica, Arial, sans-serif; margin: auto; width: 800px;} #body * {font-size: 20px; line-height: 30px;} h2 {font-size: 30px;} h3, h4, h5, h6 {font-size: 25px;} blockquote {font-family: Georgia, Times New Roman, serif; color: #888; border-left: 3px solid #888; padding-left: 30px;} a {color: #336699; text-decoration: none;} img {display: block; margin-bottom: 30px; max-width: 800px; height: auto;} iframe {display: none;} .header, .footer {margin: 0; padding: 30px 0; text-align: center;} .header {border-bottom: 1px solid #EEE; font-size: 35px; font-weight: normal; pointer-events: none;} .header a {color: #000;} .footer {border-top: 1px solid #EEE; font-size: 15px; cursor: default;} #body {padding: 30px 0;}//--></style></head>
```

[eml.zip](https://github.com/RainLoop/rainloop-webmail/files/1284890/eml.zip)

**Comment by RainLoop** [comment]
RainLoop doesn't parse inline styles. There have been several attempts to do this, but unfortunately this has led to a variety of visual defects.

But, you may try to use [convert-headers-styles](https://github.com/RainLoop/rainloop-webmail/tree/master/plugins/_depricated/convert-headers-styles) plugin. It does exactly what you want.

**Comment by RainLoop** [comment]
![image](https://user-images.githubusercontent.com/5530092/30786106-7a1ea504-a179-11e7-9dd2-57e67e32c56d.png)

**Comment by megadr01d** [comment]
Thanks a lot for the plugin reference.
I'm gonna try to ask my provider to install it. If not, I'm pretty confident I can modify rss2email to link to an external css file instead of processing it inline.

Since the cause of this issue is identified, I'm closing.

**Comment by RainLoop** [comment]
@megadr01d 

> If not, I'm pretty confident I can modify rss2email to link to an external css file instead of processing it inline.

Yes, most of these programs use inline styles to ensure normal viewing of emails.

**Comment by megadr01d** [comment]
Also as extra information, I just realized that the commented out stylesheet is actually [part of rss2email](https://github.com/turbodog/rss2email/blob/686d5a3bee38f9aa8d6b6e0d6b221d926d020620/rss2email.py#L729). I didn't notice this before and I thought it was post-processed by Rainloop.

So, wondering how Roundcube and Mail (OS X) display the stylesheet correctly even when commented out, I found that it's a trick to [support ancient browsers](https://stackoverflow.com/questions/13219650/html-comments-tags-in-internal-stylesheet).

Simply put, old browsers, which don't support the `<style>` tag, just ignore it because it's commented out, while new browsers do the opposite, and ignore the comment tag (`<!-- -->`) surrounding an inline stylesheet.

**Comment by Oliv4945** [comment]
Hi @RainLoop, I have the same issue.
The plugin `convert-headers-styles` is now deprecated, so I can not install it from the admin page. What is the best way to install it ?

Thank you

**Comment by megadr01d** [comment]
Can't you upload the `convert-headers-styles` dir directly into the `plugins` dir?
